### PR TITLE
Merge release/version2.3 into master

### PR DESCRIPTION
**Pull request by matthpeterson**

### CODE REVIEW

File: warehouse/ingest-scripts/src/main/resources/bin/ingest/bulk-ingest.sh [hunk 1]
```diff
@@ -65,9 +65,9 @@ export HADOOP_OPTS="-Dfile.encoding=UTF8 -Duser.timezone=GMT $HADOOP_INGEST_OPTS
...
+echo $INGEST_HADOOP_HOME/bin/hadoop jar ${DATAWAVE_INGEST_CORE_JAR} datawave.ingest.mapreduce.job.IngestJob -jt $INGEST_JOBTRACKER_NODE $INPUT_FILES ${INGEST_CONFIG[@]} -cacheBaseDir $JOB_CACHE_DIR -cacheJars $LIBJARS -user $USERNAME -pass $PASSWORD -instance $WAREHOUSE_INSTANCE_NAME -zookeepers $WAREHOUSE_ZOOKEEPERS -workDir $WORKDIR  -flagFileDir ${FLAG_DIR} -flagFilePattern '.*_(bulk)_.*\.flag' -srcHdfs $INGEST_HDFS_NAME_NODE -destHdfs $WAREHOUSE_HDFS_NAME_NODE -distCpConfDir $WAREHOUSE_HADOOP_CONF -mapred.map.child.java.opts=\"$CHILD_MAP_OPTS\" -mapred.reduce.child.java.opts=\"$CHILD_REDUCE_OPTS\" "${BATCHWRITER_OPTS}" $MAPRED_OPTS $EXTRA_OPTS
```
Comment (matthpeterson) [inline warehouse/ingest-scripts/src/main/resources/bin/ingest/bulk-ingest.sh]: @hlgp and @tomnelson, this PR is a merge from version2.3 to master.  Can you confirm the changes related to your work?

File: warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+COMMONS_COLLECTIONS_JAR=$(findJar commons-collections4)
+COMMONS_CONFIGURATION_JAR=$(findJar commons-configuration)
+AVRO_JAR=$(findJar avro)
+HTTPCLIENT_JAR=$(findJar httpclient)
+HTTPCORE_JAR=$(findJar httpcore)
+COMMONS_IO_JAR=$(findJar commons-io)
+COMMONS_POOL_JAR=$(findJar commons-pool)
+COMMONS_JCI_CORE_JAR=$(findJar commons-jci-core)
+COMMONS_JCI_FAM_JAR=$(findJar commons-jci-fam)
+GUAVA_JAR=$(findJar guava)
```
Comment (tomnelson) [inline warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh]: If this is replacing ingest-libs.sh, you will need:
CAFFEINE_JAR=$(findJar caffeine)

Comment (matthpeterson) [reply warehouse/ingest-scripts/src/main/resources/bin/ingest/findJars.sh]: Thanks for catching that.  I'm going to close this and create a new PR that accounts for those types of changes.

**Comment by matthpeterson** [comment]
See https://github.com/NationalSecurityAgency/datawave/pull/270

**Pull request closed without merging**